modules/arm_plugin/armplg_build.sh

```diff
@@ -1,14 +1,18 @@
 #!/bin/sh
 
+set -x
```

Comment:
Do we need shell debugging enabled during execution of the script?

Reply:
I think it's useful and it doesn't bring many logs in the same time.
Do you prefer not to use it?

Reply:
Both options are fine for me since there anyway a lot of logs produced by the script. Just to ensure the change was intended.